.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,15 @@
+name: Run unit test
+on: pull_request
```

Comment:
I'm really sorry Brave. It should come back. It's my fault. I think `push` is better than `pull_request` too.

Reply:
Never mind. It is super simple!

Reply:
I updated!